extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -496,6 +496,28 @@ describes.realWin('amp-analytics', {
     });
   });
 
+  it.skip('engaged time with named timer functions correctly', function() {
```

Comment:
The reason the test doesn't work is that the analytics json config is misconfigured. 
Try 
```
const analytics = getAnalyticsTag({
      'requests': {'heartbeat': 'https://example.com?tt=${totalEngagedTime}&inc=${incrementalEngagedTime(bar)}'},
      'triggers': {
        'heartbeat': {
          'on': 'timer',
          'timerSpec': {
            'interval': 1,
          },
          'request': 'heartbeat',
        },
      },
    });
```

But, let's not add the test here. 
Please refer to what we do with testing `TOTAL_ENGAGED_TIME`, and add one test to `test-url-replacements.js` to make sure `INCREMENTAL_ENGAGED_TIME` will be recognized and replaced. Then another test to `test-activity.js` to make sure the correct value is returned. Thanks.